types/events.go

```diff
@@ -223,6 +228,15 @@ type EventDataStateSyncStatus struct {
 // TypeTag implements the required method of jsontypes.Tagged.
 func (EventDataStateSyncStatus) TypeTag() string { return "tendermint/event/StateSyncStatus" }
 
+type EventDataEvidenceValidated struct {
+	Evidence Evidence `json:"evidence"`
+
+	Height int64 `json:"height,string"`
```

Comment:
Would this be the height of the evidence or the height we received the evidence?

Reply:
The height of the evidence is already part of the `Evidence` struct

Reply:
It is the height of the evidence at the time it is validated.

Reply:
When I tested it, the string representation of evidence did not emit the height and this seemed cleaner to parse from a users' point of view.

Reply:
That's correct.